### PR TITLE
Add reporting-guide

### PR DESCRIPTION
**Pull request by rayandas**

### CODE REVIEW

File: src/reporting-guide.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+slug: reporting-guide
```
Comment (anistark) [inline src/reporting-guide.md]: add `layout: content.liquid`

**Review by anistark — CHANGES_REQUESTED** [review]

File: src/reporting-guide.md (hunk 1 above)
Comment (rayandas) [reply src/reporting-guide.md]: Added.

File: src/reporting-guide.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+layout: content.liquid
+title: Reporting Guide
+---
+
+At PyCon India 2024, we are committed to providing a safe and welcoming environment for all attendees. We take incidents of harassment, misconduct, or violations of our [Code Of Conduct](../code-of-conduct) very seriously. This Reporting Guide aims to provide clear instructions on how to report any concerns and the steps we will take to address reported incidents.
```
Comment (sayanchowdhury) [inline src/reporting-guide.md]:
```suggestion
[Code Of Conduct](https://in.pycon.org/2024/code-of-conduct/) 
```

File: src/reporting-guide.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+### Reporting Process
+
+If you experience any harassment, or unfair treatment, or witness any behavior you believe violates the PyCon India [Code Of Conduct](../code-of-conduct), we encourage you to report it as soon as possible. Reports can be made through the following channels:
```
Comment (sayanchowdhury) [inline src/reporting-guide.md]: use absolute URL as mentioned earlier.

File: src/reporting-guide.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+a. **Code of Conduct (CoC) Workgroup Area:** You can directly approach the CoC workgroup area with any concerns you may have. If you are unable to locate the CoC workgroup area, please ask one of our volunteers wearing "PyCon India Volunteer" badges for assistance. They will gladly direct you to the CoC Workgroup area.
```
Comment (sayanchowdhury) [inline src/reporting-guide.md]: s/PyCon India Volunteer/Volunteer/

File: src/reporting-guide.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Is the incident ongoing?:** Let us know if the incident is ongoing or if there is an immediate threat to someone's safety.
+
+### Confidentiality
+
+All reports will be treated with strict confidentiality. We understand the sensitive nature of such matters and will protect the privacy of all parties involved to the best of our ability. In some cases, we may need to make a public statement, but the identities of victims and reporters will remain confidential unless instructed otherwise.
+
+### Reporting Concerns about Volunteers or Organizers
+
+If your report concerns a possible violation by a volunteer or organizer, you can make your report directly to the Chair of the Code Of Conduct Committee, whose contact information will be made available during the conference.The current Chair is Nivesh Krishna, and you can reach out via email at vniveshkrishna.98 at gmail dot com
```
Comment (sayanchowdhury) [inline src/reporting-guide.md]:
```suggestion
If your report concerns a possible violation by a volunteer or organizer, you can make your report directly to the Chair of the Code Of Conduct Committee, whose contact information will be made available during the conference.The current Chair is Sayan Chowdhury, and you can reach out via email at 911 AT yudocaa DOT in
```

File: src/reporting-guide.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+### Response and Resolution
+
+Once a report is received, the Code Of Conduct Committee will promptly initiate an investigation. The Diversity working group will review the incident, consider all the information provided, and determine the appropriate response. Possible responses may include but are not limited to:
```
Comment (sayanchowdhury) [inline src/reporting-guide.md]:
```suggestion
Once a report is received, the Code Of Conduct Committee will promptly initiate an investigation. The Code of Conduct Working Group will review the incident, consider all the information provided, and determine the appropriate response. Possible responses may include but are not limited to:
```

File: src/reporting-guide.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+- No violation found: If the incident does not violate the [Code Of Conduct](../code-of-conduct).
```
Comment (sayanchowdhury) [inline src/reporting-guide.md]: absolute URL

**Comment by abhishekmishragithub** [comment]
Proofread, LGTM :shipit:

**Review by sayanchowdhury — APPROVED** [review]

File: src/reporting-guide.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+### Reporting Process
+
+If you experience any harassment, or unfair treatment, or witness any behavior you believe violates the PyCon India [Code Of Conduct](https://in.pycon.org/2024/code-of-conduct/), we encourage you to report it as soon as possible. Reports can be made through the following channels:
+
+- **Code of Conduct (CoC) Workgroup Area:** You can directly approach the CoC workgroup area with any concerns you may have. If you are unable to locate the CoC workgroup area, please ask one of our volunteers wearing Volunteer badges for assistance. They will gladly direct you to the CoC Workgroup area.
+
+- **Email:** You can send an email to [coc@in.pycon.org](mailto:coc@in.pycon.org) to submit your report electronically.
+
+- **In-person:** If you prefer to discuss the matter in-person, please notify any member of the PyCon India organizing team or the Code Of Conduct Committee.
```
Comment (SaptakS) [inline src/reporting-guide.md:14]: I think to write this line, we need to mention somewhere in this page who are the ones in the CoC Committee?

Comment (rayandas) [reply src/reporting-guide.md:14]: Yeah adding the COC Committee members' contact details.

**Review by anistark — APPROVED** [review]

**Pull request merged**